pkg/agent/bakerapi.go

```diff
@@ -130,3 +140,23 @@ func (agentBaker *agentBakerImpl) GetDistroSigImageConfig(
 
 	return allDistros, nil
 }
+
+func (agentBaker *agentBakerImpl) GetCachedVHDImages() ([]string, error) {
+	pwd, err := os.Getwd()
+	if err != nil {
+		return nil, fmt.Errorf("error getting current working directory: %w", err)
+	}
+	manifestFilePath := filepath.Join(pwd, "../../parts/linux/cloud-init/artifacts/manifest.json")
+
+	data, err := os.ReadFile(manifestFilePath)
```

Comment:
could we just read the contents of the file once at startup since that also wouldn't change at all during runtime?

Reply:
embedding might be useful here